app/routes/index.js

```diff
@@ -112,6 +113,18 @@ router.get('/item/:id/metadata/:metadataKey', async (req, res) => {
   getMetadataResponse(id, req.params.metadataKey, res)
 })
 
+router.get('/members', async (req, res) => {
+  try {
+    const members = await getMembers()
+    res.status(200).json({ members: members })
```

Comment:
Isn't all the complexity further down to make sure that this could be returned as an array? Has this not made all that rather unnecessary?

Reply:
changed to `res.status(200).json(members)` and now just returns an array